ast_arginfo.h

```diff
@@ -59,3 +59,13 @@ static const zend_function_entry class_ast_Node_methods[] = {
 	ZEND_ME(ast_Node, __construct, arginfo_class_ast_Node___construct, ZEND_ACC_PUBLIC)
 	ZEND_FE_END
 };
+
+static zend_class_entry *register_class_ast_Node(void)
```

Comment:
Why this change?

Reply:
I forgot to revert it; I assume gen_stubs.php wouldn't be guaranteed to be compatible with older php versions in the future.